Revit_Core_Engine/Revit_Core_Engine.csproj

```diff
@@ -12,6 +12,7 @@
     <AssemblyName>Revit_Core_Engine_2018</AssemblyName>
     <TargetFrameworkVersion>v4.5.2</TargetFrameworkVersion>
     <FileAlignment>512</FileAlignment>
+    <TargetFrameworkProfile />
```

Comment:
Same as above: please revert the changes in this file, it is irrelevant to this PR